www/js/app.js

```diff
@@ -1521,17 +1521,11 @@ function displayFileSelect () {
 
     // Set the main drop zone
     if (!params.disableDragAndDrop) {
-        configDropZone.addEventListener('dragover', handleGlobalDragover);
-        configDropZone.addEventListener('dragleave', function () {
-            configDropZone.style.border = '';
-        });
-        // Also set a global drop zone (allows us to ensure Config is always displayed for the file drop)
-        globalDropZone.addEventListener('dragover', function (e) {
-            e.preventDefault();
-            if (configDropZone.style.display === 'none') document.getElementById('btnConfigure').click();
-            e.dataTransfer.dropEffect = 'link';
-        });
+        // do globally, since the whole page functions as a drop zone, so indicate properly
```

Comment:
This comment implies the dev reading it will have knowledge of the deleted comments/code above. I suggest you start `// Set a global drop zone, so that the whole page is enabled for drag-and-drop`. I'm not sure what "so indicate properly" refers to, so I think you could delete.

Reply:
Okay I'll go back and try and improve the comprehensiveness